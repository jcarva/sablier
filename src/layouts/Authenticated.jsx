/* @flow */
import { Outlet } from 'react-router-dom';
import { LayoutWrapper, ContentWrapper } from './common';
import Header from '../components/navigation/header/Header';
import Sidebar from '../components/navigation/sidebar/Sidebar';
import { type Component } from '../types';

const Authenticated = (): Component => {
  return (
    <LayoutWrapper>
      <Header />
      <Sidebar />
      <ContentWrapper pl='70px'>
        <Outlet />
      </ContentWrapper>
    </LayoutWrapper>
  );
};

export default Authenticated;
