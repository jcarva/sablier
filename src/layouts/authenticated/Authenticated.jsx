/* @flow */
import { Outlet } from 'react-router-dom';
import Header from '../../components/navigation/header/Header';
import Sidebar from '../../components/navigation/sidebar/Sidebar';
import { type Component } from '../../types';

const Authenticated = (): Component => {
  return (
    <div style={{ height: '100%' }}>
      <Header />
      <Sidebar />
      <div style={{ paddingLeft: '70px', paddingTop: '60px', height: '100%' }}>
        <Outlet />
      </div>
    </div>
  );
};

export default Authenticated;
