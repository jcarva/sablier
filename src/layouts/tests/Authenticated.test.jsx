/* @flow */
import { render } from '@testing-library/react';
import Authenticated from '../Authenticated';

describe('Authenticated', () => {
  it('[snapshot]', () => {
    const { container } = render(<Authenticated />);
    expect(container.firstChild).toMatchSnapshot();
  });
});
