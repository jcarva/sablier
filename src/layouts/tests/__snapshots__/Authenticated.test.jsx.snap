// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Authenticated [snapshot] 1`] = `
<div
  style={
    Object {
      "height": "100%",
    }
  }
>
  <div
    className="header"
  >
    Header
  </div>
  <div
    className="sidebar"
  >
    Sidebar
  </div>
  <div
    style={
      Object {
        "height": "100%",
        "paddingLeft": "70px",
        "paddingTop": "60px",
      }
    }
  />
</div>
`;
